### ISSUE TITLE
JAWS with Internet Explorer 11 doesn't read out number of results available

### ISSUE THREAD

**Issue by accessiblewebuk** [alphagov/govuk-country-and-territory-autocomplete#28]
JAWS with Firefox does read out the number of results, but it doesn't work with Internet Explorer. Tested with JAWS 17 and JAWS 18 using IE11 on Windows 10.
